src/translations/translations.en.js

```diff
@@ -956,6 +956,50 @@ export const translations = {
   'cc-notice.icon-alt.success': `Success`,
   'cc-notice.icon-alt.warning': `Warning`,
   //#endregion
+  //#region cc-oauth-consumer
+  'cc-oauth-consumer.auth': `Your authorizations`,
```

Comment:
nitpick: we should have a more precise i18n key. Also, are we talking about authorizations or rights?